ext/handlers/filters/message/message.go

```diff
@@ -47,6 +49,18 @@ func ForwardFromChatID(id int64) filters.Message {
 	}
 }
 
+func Regex(p string) filters.Message {
+	return func(m *gotgbot.Message) bool {
+		r, err := regexp.Compile(p)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
This will cause the program to panic. That is definitely not the intention here. The programmer needs to handle the error that gets returned - if they don't, that's their choice, but killing the program isn't the answer.